spec/HtmlWebpackPluginSpec.js

```diff
@@ -486,4 +486,20 @@ describe('HtmlWebpackPlugin', function() {
       /<script src="common_bundle.js"[\s\S]*<script src="index_bundle.js">/], null, done);
   });
 
+  it('adds a favicon', function(done) {
+    testHtmlPlugin({
+      entry: path.join(__dirname, 'fixtures/favicon.js'),
+      output: {
+        path: OUTPUT_DIR,
+        filename: 'index_bundle.js'
+      },
+      module: {
+        loaders: [{ test: /\.ico$/, loader: 'file' }]
+      },
+      plugins: [
+        new HtmlWebpackPlugin({favicon:true})
+      ]
+    }, [/<link rel="shortcut icon" href="0a31c912c8b55c756f7a969982b1ff91.ico">/], null, done);
```

Comment:
I'm really bad at regex, that'd probably be cleaner than adding the has to the test...